cmd/agent/common/common_windows.go

```diff
@@ -21,7 +21,7 @@ import (
 
 var (
 	// PyChecksPath holds the path to the python checks from integrations-core shipped with the agent
-	PyChecksPath = filepath.Join(_here, "..", "checks.d")
+	PyChecksPath = filepath.Join(_here, "..", "agent", "checks.d")
```

Comment:
just a quick note: on Linux we ship the integrations-core checks in `/opt/datadog-agent/checks.d` directly (instead of `/opt/datadog-agent/agent/checks.d`). The `move` happens in the `finalize` omnibus software def: https://github.com/DataDog/datadog-agent/blob/70b2811b4a18c19b0f9888a2a96522b93e299d89/omnibus/config/software/datadog-agent-finalize.rb#L38

For cross-platform consistency I think we should try to do the same thing on windows, i.e. putting the integrations-core checks in `<install_dir>/checks.d` instead of `<install_dir>/agent/checks.d`

Reply:
OK... but I'll do that in a separate PR.